We do not have these (mostly optional) pytest plugins packaged yet.

--- pydantic-2.10.6/pyproject.toml.orig
+++ pydantic-2.10.6/pyproject.toml
@@ -72,12 +72,8 @@
     'eval-type-backport',
     'pytest',
     'pytest-mock',
-    'pytest-pretty',
-    'pytest-examples',
     'faker',
     'pytest-benchmark',
-    'pytest-codspeed',
-    'pytest-memray; platform_python_implementation == "CPython" and platform_system != "Windows"',
     'packaging',
     'jsonschema',
 ]
